[master] False positive: negative in script and positive in global results

I have a [project](https://github.com/Vauxoo/odoo-extra/pull/47) with this case.

shippable case
![false negative](https://cloud.githubusercontent.com/assets/6644187/7104214/d706ca3c-e095-11e4-8917-684ba8246efc.png)

travis case
![false red](https://cloud.githubusercontent.com/assets/6644187/7104201/3e64e674-e095-11e4-947b-28f464d4f74b.png)

Global result is positive.
Particular result is negative

I add a pdb sentence in next line:
https://github.com/OCA/maintainer-quality-tools/blob/master/travis/test_server#L251

Result was:
```python
(pdb) print locals()
{'odoo_full': 'odoo/odoo', 'install_options': [], 'returncode': 1, 'stdout': <closed file 'stdout.log', mode 'w' at 0x7fde714d4930>, 'to_test_list': ['runbot_cla,runbot,runbot_prebuild'], 'command_call': ['unbuffer', 'coverage', 'run', '/root/odoo-8.0/openerp-server', '-d', 'openerp_test', '--stop-after-init', '--log-level', 'info', '--addons-path', '/root/workspace/src/github.com/Vauxoo/odoo-extra,/root/runbot-addons,/root/odoo-8.0/addons', '--log-handler', 'openerp.tools.yaml_import:DEBUG', '--test-enable', '--update', 'runbot_cla,runbot,runbot_prebuild'], 'cmd_odoo': ['/root/odoo-8.0/openerp-server', '-d', 'openerp_template', '--log-level=warn', '--stop-after-init', '--addons-path', '/root/workspace/src/github.com/Vauxoo/odoo-extra,/root/runbot-addons,/root/odoo-8.0/addons', '--init', 'runbot_cla,runbot,runbot_prebuild'], 'tested_addons': 'runbot_cla,runbot,runbot_prebuild', 'tested_addons_list': ['runbot_cla', 'runbot', 'runbot_prebuild'], 'exclude_list': ['base_report_designer','crm_profiling', 'document_fs', 'plugin', 'plugin_outlook', 'plugin_thunderbird', 'pos_cache'], 'server_path':'/root/odoo-8.0', 'errors': 0, 'counted_errors': 0, 'odoo_exclude': 'base_report_designer,crm_profiling,document_fs,plugin,plugin_outlook,plugin_thunderbird,pos_cache', 'odoo_unittest': False, 'addons_path_list': ['/root/workspace/src/github.com/Vauxoo/odoo-extra', '/root/runbot-addons', '/root/odoo-8.0/addons'], 'odoo_repo': 'odoo', 'odoo_org': 'odoo', 'expected_errors': 0, 'odoo_include': None, 'all_errors': ['runbot_cla,runbot,runbot_prebuild'], 'preinstall_modules': 'runbot_cla,runbot,runbot_prebuild', 'to_test': 'runbot_cla,runbot,runbot_prebuild', 'cmd_odoo_test': ['coverage', 'run', '/root/odoo-8.0/openerp-server', '-d', 'openerp_test', '--stop-after-init', '--log-level', 'info', '--addons-path', '/root/workspace/src/github.com/Vauxoo/odoo-extra,/root/runbot-addons,/root/odoo-8.0/addons', '--log-handler', 'openerp.tools.yaml_import:DEBUG', '--test-enable', '--update','runbot_cla,runbot,runbot_prebuild'], 'server_dirname': 'odoo-8.0', 'line': 'Coverage.py warning: No data was collected.\n', 'check_loaded': True, 'commands': ((['coverage', 'run', '/root/odoo-8.0/openerp-server', '-d', 'openerp_test', '--stop-after-init', '--log-level', 'info', '--addons-path', '/root/workspace/src/github.com/Vauxoo/odoo-extra,/root/runbot-addons,/root/odoo-8.0/addons', '--log-handler', 'openerp.tools.yaml_import:DEBUG', '--test-enable', '--update', 'runbot_cla,runbot,runbot_prebuild'], True),), 'travis_home': '/root', 'test_loghandler': 'openerp.tools.yaml_import:DEBUG', 'database': 'openerp_test', 'test_loglevel': 'info', 'travis_build_dir': '/root/workspace/src/github.com/Vauxoo/odoo-extra', 'options': ['--test-enable'], 'pipe': <subprocess.Popen object at 0x7fde70331a50>, 'command': ['coverage', 'run', '/root/odoo-8.0/openerp-server', '-d', 'openerp_test', '--stop-after-init', '--log-level', 'info', '--addons-path', '/root/workspace/src/github.com/Vauxoo/odoo-extra,/root/runbot-addons,/root/odoo-8.0/addons', '--log-handler', 'openerp.tools.yaml_import:DEBUG', '--test-enable', '--update', 'runbot_cla,runbot,runbot_prebuild'], 'addons_path': '/root/workspace/src/github.com/Vauxoo/odoo-extra,/root/runbot-addons,/root/odoo-8.0/addons', 'odoo_version': '8.0', 'x': 'plugin_thunderbird', 'pdb': <module 'pdb' from '/usr/lib/python2.7/pdb.pyc'>, 'dbtemplate': 'openerp_template'}

(pdb) print globals()
{'get_addons': <function get_addons at 0x7fde70335578>, 'parse_list': <function parse_list at 0x7fde70335668>, 'main': <function main at 0x7fde70335758>, 'success_msg': '\x1b[1;32mSuccess\x1b[0;m', 'has_test_errors': <function has_test_errors at 0x7fde703355f0>, '__builtins__': <module '__builtin__' (built-in)>, 'get_modules': <function get_modules at 0x7fde714fe140>, '__file__': '/root/maintainer-quality-tools/travis/test_server', 'fail_msg': '\x1b[1;31mFAIL\x1b[0;m', '__package__': None, 'sys': <module 'sys' (built-in)>, 're': <module 're' from '/usr/lib/python2.7/re.pyc'>, 'str2bool': <function str2bool at 0x7fde703356e0>, '__name__': '__main__', 'subprocess': <module 'subprocess' from '/usr/lib/python2.7/subprocess.pyc'>, 'os': <module 'os' from '/usr/lib/python2.7/os.pyc'>, '__doc__': None, 'print_function': _Feature((2, 6, 0, 'alpha', 2), (3, 0, 0, 'alpha', 0), 65536)}

print to_test in all_errors
True

print expected_errors and counted_errors != expected_errors
0

print counted_errors != expected_errors
False

```

(Pdb) print open('stdout.log', 'r').read()
```logger
2015-04-20 20:05:18,291 767 INFO ? openerp: OpenERP version 8.0
2015-04-20 20:05:18,291 767 INFO ? openerp: addons paths: ['/root/.local/share/Odoo/addons/8.0', u'/root/workspace/src/github.com/Vauxoo/odoo-extra', u'/root/runbot-addons', u'/root/odoo-8.0/addons', '/root/odoo-8.0/openerp/addons']
2015-04-20 20:05:18,291 767 INFO ? openerp: database hostname: localhost
2015-04-20 20:05:18,292 767 INFO ? openerp: database port: 5432
2015-04-20 20:05:18,292 767 INFO ? openerp: database user: root
2015-04-20 20:05:18,430 767 INFO ? openerp.service.server: HTTP service (werkzeug) running on 0.0.0.0:8069
2015-04-20 20:05:18,440 767 INFO openerp_test openerp.modules.loading: loading 1 modules...
2015-04-20 20:05:18,448 767 INFO openerp_test openerp.modules.loading: 1 modules loaded in 0.01s, 0 queries
2015-04-20 20:05:20,340 767 INFO ? openerp.addons.bus.bus: Bus.loop listen imbus on db postgres
2015-04-20 20:05:20,748 767 INFO openerp_test openerp.addons.report.models.report: Will use the Wkhtmltopdf binary at /usr/bin/wkhtmltopdf
2015-04-20 20:05:20,887 767 INFO openerp_test openerp.addons.report.models.report: Upgrade Wkhtmltopdf to (at least) 0.12.0
2015-04-20 20:05:21,008 767 INFO openerp_test openerp.http: HTTP Configuring static files
2015-04-20 20:05:21,327 767 INFO openerp_test openerp.modules.loading: loading 31 modules...
2015-04-20 20:05:21,863 767 INFO openerp_test openerp.modules.module: module runbot: creating or updating database tables
2015-04-20 20:05:21,988 767 INFO openerp_test openerp.modules.loading: loading runbot/runbot.xml
2015-04-20 20:05:22,407 767 INFO openerp_test openerp.modules.loading: loading runbot/res_config_view.xml
2015-04-20 20:05:22,458 767 INFO openerp_test openerp.modules.loading: loading runbot/security/runbot_security.xml
2015-04-20 20:05:22,672 767 INFO openerp_test openerp.modules.loading: loading runbot/security/ir.model.access.csv
2015-04-20 20:05:22,718 767 INFO openerp_test openerp.modules.loading: loading runbot/security/ir.rule.csv
2015-04-20 20:05:23,460 767 INFO openerp_test openerp.modules.module: module runbot_cla: creating or updating database tables
2015-04-20 20:05:23,837 767 INFO openerp_test openerp.modules.module: module runbot_language: creating or updating database tables
2015-04-20 20:05:23,890 767 INFO openerp_test openerp.modules.loading: loading runbot_language/view/runbot_language_view.xml
2015-04-20 20:05:24,427 767 INFO openerp_test openerp.modules.module: module runbot_pylint: creating or updating database tables
2015-04-20 20:05:24,466 767 INFO openerp_test openerp.modules.loading: loading runbot_pylint/view/runbot_pylint_view.xml
2015-04-20 20:05:25,004 767 INFO openerp_test openerp.modules.module: module runbot_prebuild: creating or updating database tables
2015-04-20 20:05:25,396 767 INFO openerp_test openerp.modules.loading: loading runbot_prebuild/security/runbot_team_security.xml
2015-04-20 20:05:25,730 767 INFO openerp_test openerp.modules.loading: loading runbot_prebuild/security/ir.model.access.csv
2015-04-20 20:05:25,865 767 INFO openerp_test openerp.modules.loading: loading runbot_prebuild/wizard/wizard_kill_builds.xml
2015-04-20 20:05:25,913 767 INFO openerp_test openerp.modules.loading: loading runbot_prebuild/runbot_prebuild_view.xml
2015-04-20 20:05:26,674 767 INFO openerp_test openerp.modules.loading: 31 modules loaded in 5.35s, 0 queries
2015-04-20 20:05:27,273 767 INFO openerp_test openerp.modules.loading: Modules loaded.
2015-04-20 20:05:27,308 767 INFO openerp_test openerp.modules.module: openerp.addons.base.tests.test_xmlrpc running tests.
2015-04-20 20:05:27,308 767 INFO openerp_test openerp.addons.base.tests.test_xmlrpc: test_01_xmlrpc_login (openerp.addons.base.tests.test_xmlrpc.test_xmlrpc)
2015-04-20 20:05:27,308 767 INFO openerp_test openerp.addons.base.tests.test_xmlrpc: ` Try to login on the common service.
2015-04-20 20:05:27,423 767 INFO openerp_test openerp.service.common: successful login from 'admin' using database 'openerp_test'
2015-04-20 20:05:27,423 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:27] "POST /xmlrpc/2/common HTTP/1.1" 200 -
2015-04-20 20:05:27,424 767 INFO openerp_test openerp.addons.base.tests.test_xmlrpc: test_xmlrpc_ir_model_search (openerp.addons.base.tests.test_xmlrpc.test_xmlrpc)
2015-04-20 20:05:27,424 767 INFO openerp_test openerp.addons.base.tests.test_xmlrpc: ` Try a search on the object service.
2015-04-20 20:05:27,529 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:27] "POST /xmlrpc/2/object HTTP/1.1" 200 -
2015-04-20 20:05:27,534 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:27] "POST /xmlrpc/2/object HTTP/1.1" 200 -
2015-04-20 20:05:27,536 767 INFO openerp_test openerp.addons.base.tests.test_xmlrpc: Ran 2 tests in 0.228s
2015-04-20 20:05:27,536 767 INFO openerp_test openerp.addons.base.tests.test_xmlrpc: OK
2015-04-20 20:05:27,541 767 INFO openerp_test openerp.modules.module: openerp.addons.website.tests.test_crawl running tests.
2015-04-20 20:05:27,542 767 INFO openerp_test openerp.addons.website.tests.test_crawl: test_10_crawl_public (openerp.addons.website.tests.test_crawl.Crawler)
2015-04-20 20:05:27,544 767 INFO openerp_test openerp.addons.website.tests.test_crawl: Anonymous Coward /
2015-04-20 20:05:27,550 767 INFO openerp_test openerp.addons.base.ir.ir_http: Generating routing map
2015-04-20 20:05:27,637 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:27] "GET / HTTP/1.1" 302 -
2015-04-20 20:05:27,805 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:27] "GET /runbot HTTP/1.1" 200 -
2015-04-20 20:05:27,808 767 INFO openerp_test openerp.addons.website.tests.test_crawl: Anonymous Coward /page/homepage
2015-04-20 20:05:27,855 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:27] "GET /page/homepage HTTP/1.1" 200 -
2015-04-20 20:05:27,857 767 INFO openerp_test openerp.addons.website.tests.test_crawl: Anonymous Coward /page/website.contactus
2015-04-20 20:05:27,863 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:27] "GET /page/website.contactus HTTP/1.1" 301 -
2015-04-20 20:05:27,937 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:27] "GET /page/contactus HTTP/1.1" 200 -
2015-04-20 20:05:27,939 767 INFO openerp_test openerp.addons.website.tests.test_crawl: Anonymous Coward /page/website.aboutus
2015-04-20 20:05:27,945 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:27] "GET /page/website.aboutus HTTP/1.1" 301 -
2015-04-20 20:05:27,991 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:27] "GET /page/aboutus HTTP/1.1" 200 -
2015-04-20 20:05:27,995 767 INFO openerp_test openerp.addons.website.tests.test_crawl: public crawled 4 urls in 0.45s 287 queries, 0.113s 71.75q per request,
2015-04-20 20:05:27,996 767 INFO openerp_test openerp.addons.website.tests.test_crawl: test_20_crawl_demo (openerp.addons.website.tests.test_crawl.Crawler)
2015-04-20 20:05:28,106 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:28] "GET /login?login=demo&db=openerp_test&key=demo HTTP/1.1" 303 -
2015-04-20 20:05:28,807 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:28] "GET /web HTTP/1.1" 200 -
2015-04-20 20:05:28,808 767 INFO openerp_test openerp.addons.website.tests.test_crawl: demo /
2015-04-20 20:05:28,820 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:28] "GET / HTTP/1.1" 302 -
2015-04-20 20:05:28,828 767 WARNING openerp_test openerp.addons.base.ir.ir_model: Access Denied by ACLs for operation: read, uid: 4, model: runbot.team
2015-04-20 20:05:28,830 767 WARNING openerp_test openerp.addons.website.models.ir_http: 403 Forbidden:

Traceback (most recent call last):
  File "/root/odoo-8.0/addons/website/models/ir_http.py", line 196, in _handle_exception
    response = super(ir_http, self)._handle_exception(exception)
  File "/root/odoo-8.0/openerp/addons/base/ir/ir_http.py", line 146, in _handle_exception
    return request._handle_exception(exception)
  File "/root/odoo-8.0/openerp/http.py", line 659, in _handle_exception
    return super(HttpRequest, self)._handle_exception(exception)
  File "/root/odoo-8.0/openerp/addons/base/ir/ir_http.py", line 172, in _dispatch
    result = request.dispatch()
  File "/root/odoo-8.0/openerp/http.py", line 677, in dispatch
    r = self._call_function(**self.params)
  File "/root/odoo-8.0/openerp/http.py", line 303, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/root/odoo-8.0/openerp/service/model.py", line 113, in wrapper
    return f(dbname, *args, **kwargs)
  File "/root/odoo-8.0/openerp/http.py", line 300, in checked_call
    return self.endpoint(*a, **kw)
  File "/root/odoo-8.0/openerp/http.py", line 796, in __call__
    return self.method(*args, **kw)
  File "/root/odoo-8.0/openerp/http.py", line 396, in response_wrap
    response = f(*args, **kw)
  File "/root/workspace/src/github.com/Vauxoo/odoo-extra/runbot_prebuild/runbot_prebuild.py", line 571, in root_windows
    team_ids = team_obj.search(cr, request.uid, [], order='name asc')
  File "/root/odoo-8.0/openerp/api.py", line 241, in wrapper
    return old_api(self, *args, **kwargs)
  File "/root/odoo-8.0/openerp/models.py", line 1639, in search
    return self._search(cr, user, args, offset=offset, limit=limit, order=order, context=context, count=count)
  File "/root/odoo-8.0/openerp/api.py", line 241, in wrapper
    return old_api(self, *args, **kwargs)
  File "/root/odoo-8.0/openerp/models.py", line 4615, in _search
    self.check_access_rights(cr, access_rights_uid or user, 'read')
  File "/root/odoo-8.0/openerp/api.py", line 241, in wrapper
    return old_api(self, *args, **kwargs)
  File "/root/odoo-8.0/openerp/models.py", line 3477, in check_access_rights
    return self.pool.get('ir.model.access').check(cr, uid, self._name, operation, raise_exception)
  File "/root/odoo-8.0/openerp/api.py", line 241, in wrapper
    return old_api(self, *args, **kwargs)
  File "<string>", line 2, in check
  File "/root/odoo-8.0/openerp/tools/cache.py", line 122, in lookup
    value = d[key] = self.method(*args, **kwargs)
  File "/root/odoo-8.0/openerp/addons/base/ir/ir_model.py", line 800, in check
    raise openerp.exceptions.AccessError(msg % msg_params)
AccessError: ('AccessError', u'Sorry, you are not allowed to access this document. Only users with the following access level are currently allowed to do that:\n- Public\n\t- Portal\n\t- Runbot/Runbot Private\n\t- Runbot/Runbot Admin\n\n(Document model: runbot.team)')

2015-04-20 20:05:28,906 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:28] "GET /runbot HTTP/1.1" 403 -
2015-04-20 20:05:28,907 767 ERROR openerp_test openerp.addons.website.tests.test_crawl: FAIL
2015-04-20 20:05:28,907 767 INFO openerp_test openerp.addons.website.tests.test_crawl: test_30_crawl_admin (openerp.addons.website.tests.test_crawl.Crawler)
2015-04-20 20:05:29,012 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET /login?login=admin&db=openerp_test&key=admin HTTP/1.1" 303 -
2015-04-20 20:05:29,292 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET /web HTTP/1.1" 200 -
2015-04-20 20:05:29,294 767 INFO openerp_test openerp.addons.website.tests.test_crawl: admin /
2015-04-20 20:05:29,305 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET / HTTP/1.1" 302 -
2015-04-20 20:05:29,480 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET /runbot HTTP/1.1" 200 -
2015-04-20 20:05:29,484 767 INFO openerp_test openerp.addons.website.tests.test_crawl: admin /page/website.themes
2015-04-20 20:05:29,491 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET /page/website.themes HTTP/1.1" 301 -
2015-04-20 20:05:29,581 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET /page/themes HTTP/1.1" 200 -
2015-04-20 20:05:29,584 767 INFO openerp_test openerp.addons.website.tests.test_crawl: admin /page/homepage
2015-04-20 20:05:29,644 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET /page/homepage HTTP/1.1" 200 -
2015-04-20 20:05:29,647 767 INFO openerp_test openerp.addons.website.tests.test_crawl: admin /page/website.contactus
2015-04-20 20:05:29,652 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET /page/website.contactus HTTP/1.1" 301 -
2015-04-20 20:05:29,731 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET /page/contactus HTTP/1.1" 200 -
2015-04-20 20:05:29,734 767 INFO openerp_test openerp.addons.website.tests.test_crawl: admin /page/website.aboutus
2015-04-20 20:05:29,739 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET /page/website.aboutus HTTP/1.1" 301 -
2015-04-20 20:05:29,802 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET /page/aboutus HTTP/1.1" 200 -
2015-04-20 20:05:29,806 767 INFO openerp_test openerp.addons.website.tests.test_crawl: admin /website/theme_change
2015-04-20 20:05:29,970 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:29] "GET /website/theme_change HTTP/1.1" 200 -
2015-04-20 20:05:29,973 767 INFO openerp_test openerp.addons.website.tests.test_crawl: admin /website/theme_change?theme_id=website.theme_amelia
2015-04-20 20:05:30,137 767 INFO openerp_test werkzeug: 127.0.0.1 - - [20/Apr/2015 20:05:30] "GET /website/theme_change?theme_id=website.theme_amelia HTTP/1.1" 200 -
2015-04-20 20:05:30,142 767 INFO openerp_test openerp.addons.website.tests.test_crawl: admin crawled 7 urls in 1.23s 572 queries, 0.176s 81.71q per request
2015-04-20 20:05:30,143 767 INFO openerp_test openerp.addons.website.tests.test_crawl: ======================================================================
2015-04-20 20:05:30,143 767 ERROR openerp_test openerp.addons.website.tests.test_crawl: FAIL: test_20_crawl_demo (openerp.addons.website.tests.test_crawl.Crawler)
2015-04-20 20:05:30,143 767 ERROR openerp_test openerp.addons.website.tests.test_crawl: Traceback (most recent call last):
2015-04-20 20:05:30,143 767 ERROR openerp_test openerp.addons.website.tests.test_crawl: `   File "/root/odoo-8.0/addons/website/tests/test_crawl.py", line 81, in test_20_crawl_demo
2015-04-20 20:05:30,143 767 ERROR openerp_test openerp.addons.website.tests.test_crawl: `     seen = self.crawl('/', msg='demo')
2015-04-20 20:05:30,143 767 ERROR openerp_test openerp.addons.website.tests.test_crawl: `   File "/root/odoo-8.0/addons/website/tests/test_crawl.py", line 38, in crawl
2015-04-20 20:05:30,143 767 ERROR openerp_test openerp.addons.website.tests.test_crawl: `     self.assertIn( code, xrange(200, 300), "%s Fetching %s returned error response (%d)" % (msg, url, code))
2015-04-20 20:05:30,143 767 ERROR openerp_test openerp.addons.website.tests.test_crawl: ` AssertionError: 403 not found in xrange(200, 300) : demo Fetching / returned error response (403)
2015-04-20 20:05:30,143 767 INFO openerp_test openerp.addons.website.tests.test_crawl: Ran 3 tests in 2.602s
2015-04-20 20:05:30,143 767 ERROR openerp_test openerp.addons.website.tests.test_crawl: FAILED
2015-04-20 20:05:30,144 767 INFO openerp_test openerp.addons.website.tests.test_crawl:  (failures=1)
2015-04-20 20:05:30,144 767 ERROR openerp_test openerp.modules.module: Module website: 1 failures, 0 errors
2015-04-20 20:05:30,147 767 INFO openerp_test openerp.modules.module: openerp.addons.report.tests.test_reports running tests.
2015-04-20 20:05:30,148 767 INFO openerp_test openerp.addons.report.tests.test_reports: test_reports (openerp.addons.report.tests.test_reports.TestReports)
2015-04-20 20:05:30,149 767 INFO openerp_test openerp.addons.report.tests.test_reports: Ran 1 test in 0.001s
2015-04-20 20:05:30,149 767 INFO openerp_test openerp.addons.report.tests.test_reports: OK
2015-04-20 20:05:30,152 767 INFO openerp_test openerp.modules.loading: All post-tested in 2.86s, 1304 queries
2015-04-20 20:05:30,153 767 INFO openerp_test openerp.service.server: Initiating shutdown
2015-04-20 20:05:30,153 767 INFO openerp_test openerp.service.server: Hit CTRL-C again or send a second signal to force the shutdown.
Coverage.py warning: No data was collected.
```

--- a/travis/test_server.py
+++ b/travis/test_server.py
@@ -282,13 +282,18 @@ def main():
             # Find errors, except from failed mails
             errors = has_test_errors(
                 "stdout.log", database, odoo_version, check_loaded)
+            if returncode != 0:
+                all_errors.append(to_test)
+                print(fail_msg, "Command exited with code %s" % returncode)
+                # If not exists errors then
+                # add an error when returcode!=0
+                # because is really a error.
+                if not errors:
+                    errors += 1
             if errors:
                 counted_errors += errors
                 all_errors.append(to_test)
                 print(fail_msg, "Found %d lines with errors" % errors)
-            if returncode != 0:
-                all_errors.append(to_test)
-                print(fail_msg, "Command exited with code %s" % returncode)
         subprocess.call(["dropdb", database])
 
     print('Module test summary')

